README.md

```diff
@@ -22,12 +22,14 @@ binaries available for Mac, Windows, Linux, and FreeBSD. Check out the
 
 By default, the Git LFS client needs a Git LFS server to sync the large files
 it manages. This works out of the box when using popular git repository
-hosting providers like GitHub, Atlassian, etc. When you host your own
+hosting providers like GitHub*, Atlassian, etc. When you host your own
 vanilla git server, for example, you need to either use a separate
 [Git LFS server instance](https://github.com/git-lfs/git-lfs/wiki/Implementations),
 or use the [custom transfer adapter](docs/custom-transfers.md) with
 a transfer agent in blind mode, without having to use a Git LFS server instance.
 
+Note: The current single file size limit for GitHub is 2GB.
```

Comment:
I think that either this should have `*` on it (which probably makes markdown parsers confused) or there should be a stronger link between the footnote mark and this note.

Reply:
I thought this might come back. I don't mind either way. Which way would you suggest?

Reply:
There isn't official support for footnotes in markdown. Something easily searchable at least.

Then again, maybe a section on limitations from common providers is warranted? There's also the monthly bandwidth limitation on GitHub at least.

Reply:
Moved to a separate _Limitations_ section. I don't know about the other limitations and other people could perhaps add those. Do you think that might be look better now?